db/seeds.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
-require_relative "seeds/initial_seed"
+Dir[Rails.root.join("db/seeds/initial_seed.rb")].each { |seed| load seed }
```

Comment:
So we make a directory object containing a single file, and for each file in that directory we load it? Eh.

How about
```
load Rails.root.join("db/seeds/initial_seed.rb")
```
and similar below?

Reply:
I honestly don't really understand what's really going on here - I copied it from the other repo. Every time I tried to make a change like what you're suggesting, it broke something 😂

Reply:
Good point, I guess I should fix other repo 😂

Reply:
https://github.com/DFE-Digital/ecf-engage-and-learn/pull/78 is what I _hope_ will work - it worked when I tried seeding the db locally

Reply:
Oh, well, it didn't. Uhh, I don't know why. Some side effect of enclosing a file in a dictionary I guess 👎 
Fine, this works, so it can probably stay.